boa3/model/builtin/interop/nativecontract/StdLib/getstdlibscripthashmethod.py

```diff
@@ -23,10 +24,10 @@ def _body(self) -> Optional[str]:
         return None
 
     @property
-    def opcode(self) -> List[Tuple[Opcode, bytes]]:
+    def _opcode(self) -> List[Tuple[Opcode, bytes]]:
         from boa3.neo.vm.type.Integer import Integer
 
-        value = STD_LIB_SCRIPT
+        value = self.script_hash
         return [
             (Opcode.PUSHDATA1, Integer(len(value)).to_byte_array() + value)
         ]
```

Comment:
If I understood it correctly, I think this method doesn't need to exist, because `ContractGetHashMethod` already has it